package.json

```
@@ -20,6 +20,10 @@
     "bump": "node ./scripts/bump.js",
     "release": "node ./scripts/release.js"
   },
+  "engines": {
+    "node": ">= 16",
+    "npm": ">= 8"
```

The npm version is probably irrelevant for us in this context
```suggestion
    "node": ">= 16"
```